tests/src/main/python/rest/tests/utilities/TestClass.py

```diff
@@ -0,0 +1,38 @@
+import sys
+import os
+from sys import path
+from os import getcwd
+path.append(getcwd() + "/../../src/main/python/rest/tests/extract")
+import swagger_client
+import unittest
+import json
+
+
+class TestClass(unittest.TestCase):
+    #Connection Details
+    configuration = swagger_client.Configuration()
+    #Check if we should read from a json file or environment variable.
+    if os.environ['FHIR_JSON']=='False':
```

Comment:
An alternate way to accomplish this check would be to see if `FHIR_JSON` has a value with `os.getenv()`. If it does, then treat is as a file and try to load it. Otherwise, use `FHIRUSER` and `FHIRPASS`.

My gut reaction is that the check as written is sort of special behavior that hides how to properly use `FHIRUSER` and `FHIRPASS` (since `FHIR_JSON` also needs to be set to exactly `False` and the only way to know this is to dig through this code).

Reply:
Maybe we can meet and chat. For whatever reason I couldn't get that to work when I was trying it! Probably something really silly I was doing wrong but if you think you know that call I will definitely use it, you are right, this was a bit of a hack... though I really don't expect it to be set to 'False' except when run locally.

Reply:
```
import os

if os.getenv("test"):
  print("HI")
```

Running the above with `python3 test-script.py` doesn't print "HI" for me locally.  Running  `test=123 python3 test-script.py` prints "HI".

Maybe something was setting the FHIR_JSON property?